tools/eslint-config-web3-base/ts-jest.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+	parser: '@typescript-eslint/parser',
+	plugins: ['@typescript-eslint', 'jest'],
+	extends: ['./ts', 'plugin:jest/recommended', 'plugin:jest/style'],
+	env: {
+		'jest/globals': true,
+	},
+	rules: {
+		'jest/valid-title': ['error'],
+		'jest/no-conditional-expect': ['error'],
+		'import/no-extraneous-dependencies': ['error', { devDependencies: true }],
+		'jest/consistent-test-it': ['error'],
+		'class-methods-use-this': ['error'],
+		'@typescript-eslint/no-magic-numbers': 'off',
+		'@typescript-eslint/unbound-method': 'off',
+		'@typescript-eslint/no-require-imports': 'off',
+		'@typescript-eslint/no-explicit-any': 'off',
+		'@typescript-eslint/no-unsafe-assignment': 'off',
+		'@typescript-eslint/no-unsafe-member-access': 'off',
+		'@typescript-eslint/no-unsafe-call': 'off',
+		'@typescript-eslint/no-unsafe-return': 'off',
```

Comment:
I think these can all be enabled until we run into enough instances where writing code like this is necessary

Reply:
I am switching these to `error` but these will become problem very quickly as we grow the tests.